QUEST_LV_0200.tsv

```diff
@@ -872,23 +872,23 @@ QUEST_LV_0200_20150317_000871	$I could have supported O'Rourke and the troops, i
-QUEST_LV_0200_20150317_000877	The Cathedral is a legacy of the first Bishop Maven and a sanctuary with a revelation. {nl}But I can't leave this place. {nl}I can't continue the holy mission like this.
-QUEST_LV_0200_20150317_000878	It is worth the long wait. {nl}Although my body is like this, I can't handle the overflowing emotions.
-QUEST_LV_0200_20150317_000879	This will be enough.
-QUEST_LV_0200_20150317_000880	I think the scripture will be good to hold the essence of souls. {nl}Unfortunately , it's on the floor of Pamaldu. {nl}
-QUEST_LV_0200_20150317_000881	Although I'm just a soul, I have offered myself to the Goddess. {nl}I want to be placed in the scripture.
-QUEST_LV_0200_20150317_000882	Are you done with the ritual? {nl}Check if this scripture can call it right.
-QUEST_LV_0200_20150317_000883	Great. {nl}Now you have to solve the secret of the Cathedral and get the revelation. {nl}
-QUEST_LV_0200_20150317_000884	Of course, hardships will follow. {nl}The burden of holy mission will push down your shoulders. {nl}
-QUEST_LV_0200_20150317_000885	But only you and the Goddess can save this world..{nl}I wanted to guide you even if I had to turn this way.
+QUEST_LV_0200_20150317_000877	$This Cathedral is a legacy of the first Bishop Maven, and a sanctuary with a revelation. {nl}But I am unable to leave this precise area. {nl}As it stands, I've no power to aid the holy mission.
+QUEST_LV_0200_20150317_000878	$It has been worth the long wait. {nl}Although my body is like this, I am overflowing with emotion right now.
+QUEST_LV_0200_20150317_000879	$This will be enough.
+QUEST_LV_0200_20150317_000880	$I believe a Scripture will hold the Spirit Essences well enough. {nl}Unfortunately, they are all at Pamaldu Groom. {nl}
+QUEST_LV_0200_20150317_000881	$Although I'm just a spirit, I have offered myself to the Goddess. {nl}I want to be placed in a Scripture.
+QUEST_LV_0200_20150317_000882	$Are you done with the ritual? {nl}Check if this Scripture can call me correctly.
+QUEST_LV_0200_20150317_000883	$Splendid. {nl}Now you must solve the secret of the Cathedral and get the revelation. {nl}
+QUEST_LV_0200_20150317_000884	$Of course, there will be hardship. {nl}The burden of the holy mission will surely weigh down on your shoulders. {nl}
+QUEST_LV_0200_20150317_000885	$But only you and the Goddess can save this world..{nl}Even knowing my body would turn out like this, I desired to guide you.
 QUEST_LV_0200_20150317_000886	{memo X}Don't worry. You just have to finish the bowl to hold me. {nl}First, gather the Essence of Soul. {nl}
-QUEST_LV_0200_20150317_000887	Oh, the bowl should be finished in safety altar. {nl}I have spent centuries without a body so I hope you get a decent scripture.
+QUEST_LV_0200_20150317_000887	$Oh, the Vessel should be completed at the Altar of Stability. {nl}I have spent centuries without a body, so I hope you get a decent Scripture.
```

Comment:
Vessel -> vessel? (to not clutter your PR with more notes, check this for other instances if you do this)